.github/workflows/test.yml

```diff
@@ -32,6 +32,8 @@ jobs:
             python-version: "3.10"
           - os: macos-latest
             python-version: "3.11"
+          - os: macos-latest
+            python-version: "3.12"
```

Comment:
```suggestion
          - os: windows-latest
            python-version: "3.12"
```
We alternate the OS systems to test more different combinations 😉

Reply:
Nice, thanks for updating it!